zipkin-gems/zipkin-tracer/lib/zipkin-tracer.rb

```diff
@@ -31,32 +32,22 @@ def initialize(app, config=nil)
       @service_name = config[:service_name]
       @service_port = config[:service_port]
 
-      scribe =
-        if config[:scribe_server] then
-          Scribe.new(config[:scribe_server])
-        else
-          Scribe.new()
-        end
-
-      scribe_max_buffer =
-        if config[:scribe_max_buffer] then
-          config[:scribe_max_buffer]
-        else
-          10
-        end
-
-      @sample_rate =
-        if config[:sample_rate] then
-          config[:sample_rate]
-        else
-          0.1
-        end
+      if config[:scribe_server]
+        scribe = config[:scribe_server] ? Scribe.new(config[:scribe_server]) : Scribe.new()
+        careless_scribe = CarelessScribe.new(scribe)
+        scribe_max_buffer = config[:scribe_max_buffer] ? config[:scribe_max_buffer] : 10
+        ::Trace.tracer = ::Trace::ZipkinTracer.new(careless_scribe, scribe_max_buffer)
+      elsif config[:zookeeper]
+        ::Trace.tracer = ::Trace::ZipkinKafkaTracer.new(config[:zookeeper])
+      end
+
+      @sample_rate = config[:sample_rate] ? config[:sample_rate] : 0.1
```

Comment:
yep, its there in the readme.md 
in the section for how to create Rails.config.zipkin_tracer config keys